internal/parser/scanner/ruleset/ruleset_default.go

```diff
@@ -23,13 +24,17 @@ var (
 	// defaultLinefeedDetector is the linefeed detector that this ruleset allows
 	defaultLinefeedDetector   = matcher.RuneWithDesc("linefeed", '\n')
 	defaultStatementSeparator = matcher.RuneWithDesc("statement separator", ';')
+	defaultDecimalPoint       = matcher.RuneWithDesc("decimalPoint", '.')
 	// defaultLiteral matches the allowed letters of a literal
 	defaultLiteral = matcher.Merge(
 		matcher.New("Upper", unicode.Upper),
 		matcher.New("Lower", unicode.Lower),
 		matcher.New("Title", unicode.Title),
 		matcher.New("Number", unicode.Number),
```

Comment:
should also be named lower case
```suggestion
		matcher.New("upper", unicode.Upper),
		matcher.New("lower", unicode.Lower),
		matcher.New("title", unicode.Title),
		matcher.New("number", unicode.Number),
```

Reply:
Sure. This wasnt changed by me tho ^^'